src/Interpreters/MySQL/tests/gtest_create_rewritten.cpp

```diff
@@ -223,3 +223,15 @@ TEST(MySQLCreateRewritten, UniqueKeysConvert)
         std::string(MATERIALIZEMYSQL_TABLE_COLUMNS) +
         ") ENGINE = ReplacingMergeTree(_version) PARTITION BY intDiv(id, 18446744073709551) ORDER BY (code, name, tenant_id, id)");
 }
+
+TEST(MySQLCreateRewritten, QueryWithColumnComments)
```

Comment:
May be you can also make integration test in `tests/integration/test_materialize_mysql_database/test.py`?